.github/workflows/ci.yml

```diff
@@ -0,0 +1,230 @@
+        run: |
+          cd /home/dependabot/dependabot-core/common && bundle exec rspec spec
+
+  composer:
+    name: Composer
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run composer tests
+        run: |
+          cd /home/dependabot/dependabot-core/composer && bundle exec rspec spec
+
+  dep:
+    name: Dep
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run dep tests
+        run: |
+          cd /home/dependabot/dependabot-core/dep && bundle exec rspec spec
+
+  docker:
+    name: Docker
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run docker tests
+        run: |
+          cd /home/dependabot/dependabot-core/docker && bundle exec rspec spec
+
+  elm:
+    name: Elm
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run elm tests
+        run: |
+          cd /home/dependabot/dependabot-core/elm && bundle exec rspec spec
+
+  git-submodules:
+    name: Git submodules
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run git submodules tests
+        run: |
+          cd /home/dependabot/dependabot-core/git_submodules && bundle exec rspec spec
+
+  go-modules:
+    name: Go modules
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run go modules tests
+        run: |
+          cd /home/dependabot/dependabot-core/go_modules && bundle exec rspec spec
+
+  gradle:
+    name: Gradle
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run gradle tests
+        run: |
+          cd /home/dependabot/dependabot-core/gradle && bundle exec rspec spec
+
+  hex:
+    name: Hex
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run hex tests
+        run: |
+          cd /home/dependabot/dependabot-core/hex && bundle exec rspec spec
+
+  maven:
+    name: Maven
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run maven tests
+        run: |
+          cd /home/dependabot/dependabot-core/maven && bundle exec rspec spec
+
+  npm-and-yarn:
+    name: npm and yarn
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run npn and yarn tests
+        run: |
+          cd /home/dependabot/dependabot-core/npm_and_yarn && bundle exec rspec spec
+
+  npm-and-yarn-js:
```

Comment:
@jurre adde this from the circle `test_core` section. I was attempting to fix this up a different PR because it uses an ancient core image 🙈

Reply:
Ah nice one, I never really understood what that check did 😬